File: api_jest/tests/api/v1/tles.ts
import axios from 'axios';
import { baseUrl } from '../constant';
import { Params } from '../types/interface';

export async function getTlesAPI(id: string, parameters: Params) {
  return await axios({
    method: 'get',
    url: `${baseUrl}/v1/satellites/${id}/tles`,
    params: parameters,
  }).catch(function (error) {
    return error.response;
  });
}

File: api_pytest/tests/api/v1/tles.py
import requests
from tests.api.types.interface import params
from tests.api.constant import base_url

def get_tles_api(id: str, parameters: params):
	return requests.request(
		'GET',
		f'{base_url}/v1/satellites/{id}/tles',
		params=parameters
	)
